matplotlib-tests/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: matplotlib-tests
+  version: 1.5.0
+
+source:
+  git_url: https://github.com/matplotlib/matplotlib.git
+  git_rev: v1.5.x
+
+requirements:
+  build:
+    - matplotlib ==1.5.0
```

Comment:
I don't like pinning to exact versions.  I get your point here, but is there a way that might require less maintenance in the future?